### PR TITLE
As user I can can upload profile picture.

### PR DESCRIPTION
**Pull request by AlinaMoskieva**
review please, @VladislavNikoshnov )

### CODE REVIEW

File: app/assets/javascripts/user_avatar.coffee [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Avatar
+  constructor: (el)->
+    @el = el
+    @$el = $(el)
+    @avatar = $(".user__avatar--large", @$el)
+    @avatarCache = $(".change-avatar-cache", @$el)
+    @changeAvatarButton = $(".change-avatar-button", @$el)
+
+    @bindings()
```
Comment (AlinaMoskieva) [inline app/assets/javascripts/user_avatar.coffee:9]:
```coffeescript
  @bindings()
  @changeAvatarFromCahce()
```

File: app/assets/javascripts/user_avatar.coffee [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  bindings: (el)->
+    @avatar.on "click", @downMenu
+    @changeAvatarButton.on "change", @changeAvatarPhoto
+    @changeAvatarFromCahce()
```
Comment (AlinaMoskieva) [inline app/assets/javascripts/user_avatar.coffee]: typo, should be changeAvatarFromCache

File: config/application.rb [hunk 3]
```diff
@@ -8,6 +8,8 @@
 
 module RailsBase
```
Comment (AlinaMoskieva) [inline config/application.rb]: rename pls

File: db/migrate/20161026125144_add_avatar_to_users.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddAvatarToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :avatar, :string
```
Comment (AlinaMoskieva) [inline db/migrate/20161026125144_add_avatar_to_users.rb:3]: i think we need to add constrain `null: false`

File: spec/features/user/upload_avatar/upload_avatar_spec.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
```
Comment (AlinaMoskieva) [inline spec/features/user/upload_avatar/upload_avatar_spec.rb]: i think we could use just `/user/upload_avatar_spec.rb`

File: spec/features/user/upload_avatar/upload_avatar_spec.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+feature "User uploads" do
+  let!(:user) { create :user }
+
+  scenario "avatar photo" do
```
Comment (AlinaMoskieva) [inline spec/features/user/upload_avatar/upload_avatar_spec.rb]: User uploads avatar

**Comment by VladislavNikoshnov** [comment]
🆗

**Pull request merged**